performance_tests/10zone1vav.idf

```diff
@@ -35,10 +35,12 @@
   Output:DebuggingData,0,0;
 
   ZoneCapacitanceMultiplier:ResearchSpecial,
+    1.,                      !- Name
```

Comment:
Very interesting.  This object has \min-fields 6 in the IDD, which is why the transition cleanup forces the extra fields on the end.  If those fields really aren't needed, we would probably rather just change the IDD entry, then it wouldn't add all these unnecessary trailing fields.